sap/5.3/modules/ROOT/pages/sap-connector-examples.adoc

```diff
@@ -169,70 +344,64 @@ Example response:
  			<CRETIM>050305</CRETIM>
  			<SERIAL>20191004050305</SERIAL>
  		</EDI_DC40>
-
  		...
-
-    ```
 ----
 
-=== XML to Send an IDoc to SAP
+== Receive an SAP Function
 
-[source,xml,linenums]
+This example shows you how to create a Mule app that receives an SAP function. The following screenshot shows the Studio flow for this example:
+
+image::sap-connector-example-establish-connection.png[Studio flow for establishing an SAP connection]
+
+To create the flow:
```

Comment:
This also requires a flow for triggering a bapi function via SRFC. The screenshot should be updated to contain both flows. And the mule application should be also updated